pages/common/docker-exec.md

```diff
@@ -22,3 +22,7 @@
 - Set an environment variable in a running bash session:
 
 `docker exec -it -e {{variable_name}}={{value}} {{container_name}} {{/bin/bash}}`
+
+- Execute command as a specific user:
+
+`docker exec -u {{user}} {{container_name}} {{command}}`
```

Comment:
Should the user be already created in the container, or is created by docker automatically?
```suggestion
`docker exec --user {{user}} {{container_name}} {{command}}`
```

Reply:
Nice question, The user must be determined in the Dockerfile. Just can pick one of the existing users in the container.

Reply:
And by this change, I think it's better to change all shortened options to the original form.